recipe/meta.yaml

```diff
@@ -58,7 +59,6 @@ outputs:
       - Library/bin/libcurl.dll  # [win]
       - Library/include/curl     # [win]
       - Library/lib/libcurl.lib  # [win]
-      - Library/lib/libcurl.exp  # [win]
```

Comment:
The CMake build no longer produces the `.exp` file. AFAICT the import `.lib` should be sufficient. So went ahead and dropped this